site/src/analytics-versions.ts

```diff
@@ -44,7 +44,7 @@ export class AnalyticsVersions extends LitElement {
         ]}
         .options=${{
           title: "Top 5 used versions",
-          chartArea: { width: "70%", height: "80%" },
+          chartArea: { width: "70%", height: "70%" },
```

Comment:
Why do we have to make the height smaller? On mobile the chart is getting really small now?

Reply:
Looked cleaner, can probably do 100% for mobile